config/quality/checkstyle/checkstyle.xml

```diff
@@ -48,7 +48,9 @@
         <module name="DeclarationOrder" />
         <module name="DefaultComesLast" />
         <module name="EmptyStatement" />
-        <module name="EqualsAvoidNull" />
+        <module name="EqualsAvoidNull" >
+            <property name="severity" value="ignore" />
```

Comment:
Can you remove this ignore and suppress the issue where you need it 😇 I do (generally) want to avoid this...